LDAP CA: fix screen (bsc#1207945)

taroth21 opened the pull request:
### PR creator: Description

Fix line break in screen.


### PR creator: Are there any relevant issues/feature requests?

* bsc#1207945.
* jsc#DOCTEAM-959


### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

taroth21 commented:
Line break is fixed and looks good in both PDF and HTML output now: 
/local/git/GH/doc-sle/build/SLES-security/cha-security-ldap_en.pdf

rhabacker commented on xml/security_ldap_ca.xml:
```diff
@@ -82,8 +82,8 @@
   </para>
-  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA   pk12util -i <replaceable>SERVER.p12</replaceable></command> \
-<command>-d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
+  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA \
```
There is a `;` missing before the `\` otherwise the two concatenated lines will still be one command.

rhabacker commented on xml/security_ldap_ca.xml:
```diff
@@ -82,8 +82,8 @@
-  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA   pk12util -i <replaceable>SERVER.p12</replaceable></command> \
-<command>-d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
+  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA \
+pk12util -i <replaceable>SERVER.p12</replaceable></command> <command>-d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
```
This command needs to be run as `root`, so it should be prefixed with `sudo`:

```
sudo pk12util -i <replaceable>SERVER.p12</replaceable></command> <command>-d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
```

taroth21 commented:
@rhabacker : Thank you! I addressed your comments with https://github.com/SUSE/doc-sle/pull/1487/commits/666f60baf049b5f3d1d970 (and double-checked against our style guide). 

I think both the `\` at the end and adding `;` are no longer needed as the second sudo prefix now makes clear that those are two separate commands.

rhabacker commented:
> I think both the `\` at the end and adding `;` are no longer needed as the second sudo prefix now makes clear that those are two separate commands.

I agree - this is better

> Updated PDF: [cha-security-ldap_2023-04-24.pdf](https://github.com/SUSE/doc-sle/files/11308523/cha-security-ldap_2023-04-24.pdf)

looks good - thanks for you work.

rhabacker commented:
> I addressed your comments with https://github.com/SUSE/doc-sle/commit/666f60baf049b5f3d1d970 

I think you can combine both commits into one.
